cmd/root.go

```diff
@@ -118,6 +120,16 @@ func NewCommand(opts ...Option) *Command {
 	return cmd
 }
 
+// wordSepNormalizeFunc normalize flag names so that - and _ is the same.
+func wordSepNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedName {
+	from := []string{"_"}
+	to := "-"
+	for _, sep := range from {
+		name = strings.Replace(name, sep, to, -1)
+	}
```

Comment:
Why do we iterate over a array with len 1?
```suggestion
	name = strings.ReplaceAll(name, "-", "_")
```